src/popup/components/Modals/AssetSelector.vue

```diff
@@ -24,19 +24,27 @@
       class="appearing-element"
       :class="{ visible: loading }"
     />
-    <div
-      v-show="isFullyOpen"
-      class="appearing-element"
-      :class="{ visible: !loading }"
-    >
-      <AssetListItem
-        v-for="asset in accountAssetsToDisplay"
-        :key="asset.contractId"
-        :asset="asset"
-        :selected="isAssetSelected(asset)"
-        prevent-navigation
-        @click="resolve(asset)"
-      />
+    <div ref="innerScrollElem">
```

Comment:
If this element wraps the infinite scroll it's not `inner` but `outer` :)

Reply:
This is a chosen class name wording in all similar situations, We can change it everywhere, if you think that it is more suitable.